hazelcast-jet-distribution/src/root/release_notes.txt

```diff
@@ -23,6 +23,70 @@ To expand an issue number into a full GitHub link, use this template:
 
 https://github.com/hazelcast/hazelcast-jet/issues/<issue number>
 
+
+** Hazelcast Jet 4.3 **
+
+Members of the open source community that appear in these release notes:
+
+@caioguedes
+@guenter-hesse
+@MohamedMandouh
+
+Thank you for your valuable contributions!
+
+1. New Features
+
+[pipeline-api] [014] @MohamedMandouh implemented distributed sorting: `BatchStage.sort()`
+  (#2469, #2544)
+[core] [012] Added `JobConfig.suspendOnFailure`: suspend a job on exception instead of
+  cancelling it (#2411)
+[cdc] Improved the consistency of reconnect behaviour across CDC sources, new uniform API
+  to configure the reconnect strategy (#2419)
+
+2. Enhancements
+
+[core] @guenter-hesse contributed a test source to benchmark Jet's throughput and latency (#2382)
+[core] [013] Improved watermark semantics that prevent low event rate from stalling an
+  ingestion time-based pipeline (#2485, #2514)
+[cdc] Exposed the sequence number in the CDC `ChangeRecord` that orders the events (#2390)
+[core] Two new DAG edge types: `distributeToOne` (sending all data to one member) and `ordered`
+  (maintaining the sort order) (#2394, #2469, #2544)
+[core] Disabled access to external XML entities when parsing XML config, this was a potential
+  XXE attack vector (#2528)
+
+3. Fixes
+
+[core] Fixed error handling during job startup that could result in inconsistent job state (#2383)
+[core] Fixed an internal exception that leaked out of Observable (#2313, #2389)
+[core] Prevented Observable from processing in-flight items after cancellation (#2415, #2418)
+[cli] @caioguedes fixed an issue with `--targets` option in CLI where it would overwrite other
+  settings (#2373, #2421)
+[metrics] Fixed a problem where an internally added DAG vertex would show up as a source instead
+  of the actual source vertex (#2475, #2476)
+[core] Fixed a race that could cause `getJobStatus()` to throw an exception if called right after
+  `newJob()` (#2481, #2484)
+[core] Fix a race between snapshotting and restarting (#2487, #2503)
+[core] Fixed a race where `getJobStatus()` would report `RUNNING` even though it was actually
+  `COMPLETING`. (#2507)
+[core] Fixed an issue where a `DONE_ITEM` could get lost due to connection failure, preventing
+  the job from completing (#2158, #2532)
+[core] Fixed a job failure related to the coordinator member failing (#2461, #2546)
+[core] Fixed a job failure related to a member reconnecting (#2542, #2547)
+[core] Improved robustness related to Jet's internal `IMap` operations (#2533, #2550)
+[cdc] Upgraded Jackson jr dep, solving a null handling issue (#2459)
```

Comment:
This was in fact a more generic, JSON handling problem, just more obvious in the context of CDC. Now, if that's worth mentioning or not, that's your call. Just saying.